app/src/main/java/io/neurolab/FakeLock.java

```diff
@@ -0,0 +1,17 @@
+package io.neurolab;
+
+public class FakeLock {
```

Comment:
What is the significance of including this class in a single PR? Any use of it in the Android application or in any classes currently?